pkg/es/config/config_test.go

```diff
@@ -345,66 +358,108 @@ func TestApplyDefaults(t *testing.T) {
 		{
 			name: "All Defaults Applied except PriorityDependenciesTemplate",
 			target: &Configuration{
-				PriorityDependenciesTemplate: 30,
+				Indices: Indices{
+					Spans:    IndexOptions{},
+					Services: IndexOptions{},
+					Dependencies: IndexOptions{
+						Priority: 30,
+					},
+					Sampling: IndexOptions{},
+				},
```

Comment:
```suggestion
				Indices: Indices{
					Dependencies: IndexOptions{
						Priority: 30,
					},
				},
```
no need to provide zero values